src/table/utils/styling-utils.ts

```diff
@@ -259,6 +258,7 @@ export function getSelectionStyle(styling: CellStyle, cellSelectionState: Select
     case SelectionStates.EXCLUDED:
       selectionStyling = {
         background: `${STYLING_DEFAULTS.EXCLUDED_BACKGROUND}, ${styling.backgroundColor}`,
+        selectedCellClass: SelectionStates.EXCLUDED,
```

Comment:
I don't think this should live on the styling. it simplifies thing but it is a bit misleading. Might refactor in a different PR